packages/client/src/ActionsAndReducers/PlayerLog/PlayerLog_Reducer.ts

```diff
@@ -0,0 +1,18 @@
+import * as ActionConstant from '@serge/config'
+import { PLayerLogAction, PlayerLogPayload } from './PlayerLog_types'
+
+const initialState: PlayerLogPayload = {
+  wargame: '',
+  role: ''
+}
+
+export const addPlayerLogReducer = (state = initialState, action: PLayerLogAction) => {
+
```

Comment:
can you remove the blank lines please @lebaphi ?

Reply:
updated